ecosystem/sep-0012.md

```diff
@@ -315,3 +316,38 @@ Situation | Response
 Success | `200 OK`
 Client not authenticated properly | `401 Unauthorized`
 Anchor has no information on the customer | `404 Not Found`
+
+## Customer callback PUT
+
+Allow the wallet to provide a callback URL to the anchor. The provided callback 
+URL will replace (and supercede) any previously-set callback URL for this account.
+
+Whenever the user's
+`status` field changes, the anchor will issue a POST request to the callback
+URL. The payload of the POST request will be the same as [`GET /customer`](#customer-get).
+
+Anchors will submit POST requests until the user's status changes to `ACCEPTED`
+or `REJECTED`. If a wallet needs to watch a user's KYC status after that,
+it will need to set a callback again.
+
+### Request
+
+```
+PUT [KYC_SERVER || TRANSFER_SERVER]/customer/callback
+```
+
+| Name                 | Type     | Description             |
+| -------------------- | -------- | ----------------------- |
+| `id`        | `string`      | (optional) The ID of the customer as returned in the response of a previous `PUT` request. If the customer has not been registered, they do not yet have an `id`.                                                                                                                                     |
+| `account`   | `G...` string | (optional) The Stellar account ID used to identify this customer. If many customers share the same Stellar account, the `memo` and `memo_type` parameters should be included as well.                                                                                                                 |
+`memo` | `string` | (optional) The memo used to create the customer record |
+`memo_type` | `string` | (optional) The type of memo used to create the customer record |
+| `url` | `string` | A valid URL. |
+
+### PUT Responses
+
+| Situation                                 | Response           |
+| ----------------------------------------- | ------------------ |
+| Success                                   | `200 OK`           |
+| Client not authenticated properly         | `401 Unauthorized` |
```

Comment:
I think we should stick to using `403 Permission Denied` since thats used in SEP 6, 24, & 31. Maybe others as well.

Reply:
We should probably add a comment about using 403 in the authentication section since it applies to any request here.

Reply:
The other, existing calls already return 401. Are you sure you want to change the error response if this SEP is already in use?

Reply:
Ah you're right, darn. I don't want to make any breaking changes so lets just stick with 401.